structs/pokemon.h

```diff
@@ -89,7 +96,9 @@ typedef enum {
     UNOWN_W=176,
     UNOWN_X=184,
     UNOWN_Y=192,
-    UNOWN_Z=200
+    UNOWN_Z=200,
+    UNOWN_EXCLAMATIONMARK =208,
```

Comment:
I know I initially wrote these, but maybe add an underscore before `MARK`. Also (probably my bad too), the spaces are inconsistent. We should put 1 space before and 1 space after the equals sign.

Reply:
Should I also align the equal signs so it looks better?
```
typedef enum {
    ARCEUS_NORMAL   = 0,
    ARCEUS_FIST     = 8,
    ARCEUS_SKY      = 16,
```

Reply:
No, just use 1 space on either side of the equals signs.

Reply:
Done :)